### PR TITLE
Add contact

### PR DESCRIPTION
**Pull request by osamaashraf6**
## What does this PR do?

This PR is raised to add a contact form that Collects data by using the [Formspree service](https://formspree.io/html).

## Description of the task to be completed

- [x] This project is a portfolio at this I added the contact form by 👎 
- [x] Use Flexbox a layout tool in CSS that helps you easily position HTML elements on the page.
- [x] Use Figma a collaborative UI design tool built in the browser. 
- [x] Use the linter tool that analyzes my source code to flag programming errors, bugs, stylistic errors, and suspicious constructs.
- [x] [Formspree service](https://formspree.io/html)  that Collects data from HTML form.
 

  
## How should it be tested

Clone the repository
```
git clone https://github.com/osamaashraf6/portfolio.git
```
Navigate to the folder
```
cd portfolio
```
Checkout the branch
```
git checkout add_contact
```

#### Thanks for reviewing my PR 😊

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,8 +11,9 @@
 - CSS
 - Flexbox
 - Figma
+- Formspree service
 
-## Live Demo (if available)
+## Live Demo 
 
 [Portfolio](https://github.com/osamaashraf6/portfolio/tree/portfolio)
```
Comment (karlavdelgadof) [inline README.md:21]: **[OPTIONAL]:** For a more professional look :necktie: , and to call the attention of anyone who visits your profile, I would highly recommend including more sections :1234:  from the [Microverse Template](https://github.com/microverseinc/readme-template). In this way, you can provide a clear an precise idea of what your project is about, how it looks, who was part of it, etc.

File: static/css/index.css [hunk 2]
```diff
@@ -319,4 +324,80 @@ header {
...
+  border: 1px solid #fff;
+  color: #fff;
+  background-color: #6070ff;
+}
+
+.form__control > input[type="submit"]:focus {
+  background-color: #2230d2;
+  color: #fff;
+}
+
```
Comment (karlavdelgadof) [inline static/css/index.css]: - [ ] As shown in the [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) you chose :eyes: , the contact form has some illustrations at the bottom that are not present in your design. Please make sure to make it look as similar as possible by adding all the illustrations and arranging their positions with the [CSS background property](https://www.w3schools.com/cssref/css3_pr_background.asp) :heavy_check_mark: :framed_picture: 

**Your design:**
image

**Figma Template:**
image

File: static/css/index.css [hunk 3]
```diff
@@ -319,4 +324,80 @@ header {
...
+  outline: 0;
+  border-radius: 5px;
+  width: 100%;
+  padding: 10px 15px;
+  margin-bottom: 15px;
+}
+
+.form__control > input[type="text"]::placeholder,
+.form__control > input[type="email"]::placeholder,
+.form__control > textarea::placeholder {
```
Comment (karlavdelgadof) [inline static/css/index.css:378]: Fantastic job implementing CSS selectors! :star_struck: :clap:

Comment (karlavdelgadof) [inline static/css/index.css:378]: **[OPTIONAL]:** To match the Figma template I suggest setting the `font-family` of the inputs to the font-family provided by the design, this way you can accomplish more unity in your design :100: :heavy_check_mark: 

**Your project:**
image

**Figma Template:**
image

File: index.html [hunk 4]
```diff
@@ -370,5 +373,66 @@ <h3>LET’S CONNECT</h3>
...
+                <textarea
+                  id=""
+                  cols="30"
+                  rows="10"
+                  class="textarea"
+                  id="textarea"
+                  placeholder="Write your message here"
+                  maxlength="500"
+                  name="textarea"
+                  required
```
Comment (karlavdelgadof) [inline index.html:419]: Good job adding the input HTML validation! :heavy_check_mark:

**Review by karlavdelgadof — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @osamaashraf6  :technologist:  :wave: !

Your portfolio project is looking fantastic so far :star_struck: ! 

However, I've found some room for improvement that can help you accomplish a more readable/clean code and make your project look professional :sunglasses: 



## :sparkles: Highlights :sparkles:

- The contact form is working as expected :heavy_check_mark: 
- Great work using the right GitHub Flow :100: 
- Awesome file organization :card_file_box: 
- Commits are descriptive and meaningful :+1: 
- Project Documentation (PR, README) is professional :necktie: 
- Awesome job following the design guidelines! :100: 

Congrats on your work so far! :clap:  

## Required Changes ♻️
There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@karlavdelgadof**  in your question so I can receive the notification.
  
  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
  
  ------
  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -370,5 +373,66 @@ <h3>LET’S CONNECT</h3>
...
+              </div>
+              <div class="form__control">
+                <input
+                  type="submit"
+                  placeholder=""
+                  class="submit"
+                  id="submit"
+                  value="Get in touch"
+                />
+              </div>
```
Comment (MaganaAsiati) [inline index.html:430]: - You did a great job by adding the submit button and it working as expected but is just visible in the desktop version. Kindle makes sure the button for submitting the form is also visible in the mobile version as required by the project. check below your design compared to the Figma design  :grinning: :grinning: :grinning: 


## Your design 

image

## Figma design

image

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @osamaashraf6 , I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 1 above)
Comment (osamaashraf6) [reply README.md:21]: ### Thank you 💞 for your feedback I will make the changes and push it again to review it.☺️

File: index.html [hunk 6]
```diff
@@ -370,5 +373,66 @@ <h3>LET’S CONNECT</h3>
...
+            </form>
+          </div>
+        </div>
+      </footer>
```
Comment (mckent05) [inline index.html:434]: **[OPTIONAL]** - For semantic purposes and accessibility, I'd suggest that you don't wrap your contact section in a `footer` tag. You could use a `section` tag for this.

**Review by mckent05 — APPROVED** [review]
Hi @osamaashraf6 , 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 

## Highlights
- Good job implementing changes requested by the previous reviewer 👏 👏 

## Optional suggestions
- I have included some optional changes below, I advise that you implement these changes, I believe it would greatly improve your project 🙏 🙏 
- 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (osamaashraf6) [reply index.html:434]: Thank you ☺️ I appreciate your review you are an amazing reviewer again thank you for your effort 🎉.

File: static/css/index.css (hunk 3 above)
Comment (osamaashraf6) [reply static/css/index.css:378]: - Thanks @karlavdelgadof  🙏 for reviewing my pull request I appreciate your review I will work on this part to enhance my code thanks for advising me 👌 💯 
### Thanks again for reviewing my PR  and have a good day😊

File: index.html (hunk 5 above)
Comment (osamaashraf6) [reply index.html:430]: - Thanks  @MaganaAsiati 🙏 for reviewing my pull request I appreciate your review I will work on this part to enhance my code thanks for advising me 👌 💯 
### Thanks again for reviewing my PR  and have a good day😊

**Pull request merged**